features/0434-outofband/README.md

```diff
@@ -0,0 +1,417 @@
+# Aries RFC 0434: Out of Band Protocol 1.0
+
+- Authors: [Ryan West](ryan.west@sovrin.org), [Daniel Bluhm](daniel.bluhm@sovrin.org), Matthew Hailstone, [Stephen Curran](swcurran@cloudcompass.ca), [Sam Curren](sam@sovrin.org), [George Aristy](george.aristy@securekey.com)
+- Status: [PROPOSED](/README.md#proposed)
+- Since: 2020-03-01
+- Status Note: This RFC extracts the `invitation` messages from the [DID Exchange](../../features/0023-did-exchange/README.md) protocol (and perhaps [Connection](../../features/0160-connection-protocol/README.md)), and replaces the combined `present_proof/1.0/request` combined with the `~service` decorator to define an ephemeral (connection-less) challenge.
+- Supersedes: Invitation Message in [0160-Connections](https://github.com/hyperledger/aries-rfcs/blob/9b0aaa39df7e8bd434126c4b33c097aae78d65bf/features/0160-connection-protocol/README.md#0-invitation-to-connect) and Invitation Message in [0023-DID-Exchange](https://github.com/hyperledger/aries-rfcs/blob/9b0aaa39df7e8bd434126c4b33c097aae78d65bf/features/0023-did-exchange/README.md#0-invitation-to-exchange).
+- Start Date: 2020-03-01
+- Tags: [feature](/tags.md#feature), [protocol](/tags.md#protocol)
+- Protocol Name: Out of Band
+- Version: 1.0
+- URI: `http://didcomm.org/out_of_band/VER/MsgType`
```

Comment:
I believe we're using hypens in protocol names

Reply:
Fixed